src/mds/MDSRank.cc

```diff
@@ -1150,19 +1150,14 @@ void MDSRank::validate_sessions()
     Session *session = i.second;
     interval_set<inodeno_t> badones;
     if (inotable->intersects_free(session->info.prealloc_inos, &badones)) {
-      clog->error() << "Client session loaded with invalid preallocated "
-                          "inodes, evicting session " << *session;
-
-      // Make the session consistent with inotable so that it can
-      // be cleanly torn down
-      session->info.prealloc_inos.subtract(badones);
-
-      victims.push_back(session);
+      clog->error() << "client " << *session << " loaded with invalid preallocated inodes";
```

Comment:
I think we could improve the error message: `" loaded with preallocated inodes inconsistent with my inotable"`.